standards/technical-approach.md

```diff
@@ -43,8 +78,30 @@ Step 5 above includes the following steps to be followed by the Tech Lead/Archit
 2. If revisions are requested/necessary the document's author should make the necessary changes and record the process in a changelog to be added to the end of the document
 3. The above steps should be repeated until the document is approved
 
+### Changes After Approval
+
+During the implementation phase, it is possible that the approved approach is deemed insufficient and the technical team pivots to a different solution. In this case, it is important to update the original TAD in the "Changelog" section at the bottom of the document. Including a changelog with a record of why something didn’t work the way we expected provides helpful context to new and exisitng team members.
```

Comment:
Q: Is there specific format we should follow for the changelog section? How do we communicate this to ELT/leads about the change. (i.e. Do we need another approval for pivoting to a different approach?)

Reply:
I did update the TAD template with a Changelog section at the bottom, which right now only includes a date heading.
<img width="643" alt="Screenshot 2024-01-08 at 6 22 02 PM" src="https://github.com/NYPL/engineering-general/assets/34348332/bc07aa8f-36d7-41d7-b7e0-5aa9509cff10">

Updating the TAD's changelog should include communicating with your Tech Lead or Architect, and from there Crystal or Mike should probably be updated. I'm imagining this happening throughout the regular 1:1 meeting, so I'll add this process here. Good idea.

Reply:
Thanks for the clarifications!